doc_conf/conf.py

```diff
@@ -51,172 +37,112 @@
 # sphinxcontrib-bibtex
 bibtex_bibfiles = ["./references.bib"]
 bibtex_style = "unsrt"
 bibtex_reference_style = "author_year"
 bibtex_footbibliography_header = ""
 
-# General information about the project.
-project = "HiDimStat"
-copyright = "2024, Mind-Inria"
-author = "Mind-Inria"
-
-# The version info for the project you're documenting, acts as replacement for
-# |version| and |release|, also used in various other places throughout the
-# built documents.
-#
-# The short X.Y version.
-from hidimstat import __version__
-
-# The full version, including alpha/beta/rc tags.
-release = __version__
-
-# The language for content autogenerated by Sphinx. Refer to documentation
-# for a list of supported languages.
-#
-# This is also used if you do content translation via gettext catalogs.
-# Usually you set "language" from the command line for these cases.
-language = "en"
-
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
-
-# The name of the Pygments (syntax highlighting) style to use.
-pygments_style = "sphinx"
-
-# If true, `todo` and `todoList` produce output, else they produce nothing.
-todo_include_todos = False
 
+# -- Options for autodoc / autosummary ----------------------------------------
 # generate autosummary even if no references
 autosummary_generate = True
 
-# remove warnings: "toctree contains reference to nonexisting document"
+# -- Options for Numpydoc ----------------------------------------------------
+# https://numpydoc.readthedocs.io/en/latest/install.html
 numpydoc_show_class_members = False
 
-# -- Options for HTML output ----------------------------------------------
-
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-html_theme = "bootstrap"
+# -- Options for autodoc ----------------------------------------------------
+autodoc_default_options = {
+    "inherited-members": True,
+    "member-order": "bysource",
+    "show-inheritance": True,
+}
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 html_theme_options = {
-    "navbar_sidebarrel": False,
-    "navbar_pagenav": False,
-    "source_link_position": "",
-    "navbar_links": [
-        ("Examples", "auto_examples/index"),
-        ("API", "api"),
-        ("GitHub", "https://github.com/mind-inria/hidimstat", True),
+    "icon_links": [
+        {
+            "name": "GitHub",
+            "url": "https://github.com/mind-inria/hidimstat/",
+            "icon": "fa-brands fa-github",
+        },
     ],
-    "bootswatch_theme": "flatly",
-    "bootstrap_version": "3",
+    "navbar_end": ["version-switcher", "theme-switcher", "navbar-icon-links"],
+    "header_links_before_dropdown": 4,
 }
```

Comment:
Move it to the section HTML.
I made a suggestion for it.

Reply:
thx